addons/vehicleDamage/config.cpp

```diff
@@ -0,0 +1,20 @@
+#include "script_component.hpp"
+
+class CfgPatches {
+    class ADDON {
+        name = COMPONENT_NAME;
+        units[] = {};
+        weapons[] = {};
+        requiredVersion = REQUIRED_VERSION;
+        // ammo/vehicle config defines touch all of these
+        requiredAddons[] = { "ace_common", "ace_cookoff", "ace_dragon", "ace_hot", "ace_aircraft", "ace_ballistics", "ace_hellfire", "ace_frag" };
```

Comment:
We should just do ` EGVAR(vehicleDamage,incendiary) = x` on the ammo in these addons so we don't have to require them here

Reply:
Probably. Like I said I don't really understand *when* to use `requiredAddons`. I figured whenever you modify a config you'd want to add it in. Will fix